### ISSUE TITLE
How can i get a plot a plot of precision recall curve or ROC curve for a custom dataset?

### ISSUE THREAD

**Issue by aryu99** [AlexeyAB/darknet#6351]
I am doing a project where I am required to plot the Precision recall and ROC curve for a custom dataset. Is there any command which lets me get these plots?
I tried this :
./darknet detector map data/obj.data yolo-obj.cfg backup\yolo-obj_7000.weights -points 11

However, my output did not give me any plot and nor could I see any file saved in the darknet directory.
![image](https://user-images.githubusercontent.com/40424189/88477846-f73cfa00-cf60-11ea-9491-105285806329.png)
I am new to this and any help would be appreciated! :)

**Comment by A-Sassi** [comment]
@aryu99 did you figured it out?

**Comment by aryu99** [comment]
@A-Sassi Nope. But I did find a way around it. You can use the "-conf_thresh" flag in the "map" command to get precision and recall values for different confidence thresholds. You can use that to plot the PR curve.
